data/tribes/initialization/frisians/units.lua

```diff
@@ -1961,6 +1968,15 @@ wl.Descriptions():new_tribe {
             note = pgettext("frisians_building", "The farm needs free space within the work area to plant seeds.")
          }
       },
+      {
+         name = "frisians_dikers_house",
+         helptexts = {
+            -- TRANSLATORS: Purpose helptext for a frisian production site: Diker's House
+            purpose = pgettext("frisians_building", "Constructs breakwaters nearby to gain new land from the sea."),
+            -- TRANSLATORS: Note helptext for a frisian production site: Diker's House
+            note = pgettext("frisians_building", "The diker will terraform the land around each breakwater he builds several times. You can manually remove breakwaters when the land is sufficiently secured to prevent wasting resources.")
```

Comment:
1. May we add a sentence how to remove the dikes (i.e. by building a flag building or road over them)
2. while trying to understand the help bug I realized that the dike is missing as a tribe immovable in this file so we should add it and give the same hint in the immovable help.